opentelemetry-resource-detectors/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## vNext
 
+### Changed
+
+- Expose `K8sResourceDetector`, which populates `k8s.pod.name` and `k8s.namespace.name`
```

Comment:
I am curious about populating just these 2 attributes... Is that just a starting point, or we have limitations about capturing some other attributes from K8s?

Reply:
yeah, correct. These two are the easiest to retrieve without needing some adjustments of the Pod permissions. I planed to tackle the other ones in the future.

Reply:
I'll create a separate issue, which you can assign to me